privacyidea/lib/tokenclass.py

```diff
@@ -2026,3 +2042,19 @@ def import_token(self, token_information: dict):
         self.add_tokeninfo_dict(token_information.setdefault("tokeninfo", {}))
         self.add_tokeninfo("import_date", datetime.now(timezone.utc).isoformat(timespec="seconds"))
         self.save()
+
+    def assign_user_during_token_import(self, user: dict):
```

Comment:
We already have `assign_token()` and `unassign_token()` in `token.py`
Please use/adjust existing functions to avoid duplicating code.